src/contexts/AuthProvider.js

```diff
@@ -1,13 +1,10 @@
-import React, { useContext, useState, useEffect } from 'react';
-import { auth, db, googleProvider } from './FirebaseService';
+import React, { useState, useEffect } from 'react';
+import { auth, db, googleProvider } from 'services/FirebaseService';
 
```

Comment:
group imports like this and sort them alphabetically by their path

react
//space
3rd party imports
//space
in project absolute imports
//space
in project relative imports